### DISCUSSION TITLE
depth maps coming up black but not for all models.

### DISCUSSION THREAD

**Discussion by mcornetto** [thygate/stable-diffusion-webui-depthmap-script#469] [Q&A]
models like res101 and depthAnywhere base are working but other models like zoedepth and even depthAnywhere (not base) are just returning a black image.  These models were working and then suddenly stopped.  Is there a way to reinstall the models.

**Comment by sonakravtsenko3** [comment]
I highly doubt that redownloading models would help - they are basically a bunch of numbers. Please try somehow to restore the previous state and post all the python packets installed with all the versions.

Very small but not negligible probability of a GPU driver issue: please try to use CPU and see if it would fix the issue.

**Comment by mcornetto** [comment]
@semjon00 CPU is working.  Is it possible that the cache in my GPU needs to be cleared?  I already uninstalled and reinstalled the requirements and pytorch.

**Comment by sonakravtsenko3** [comment]
Alright, that's a clue! What is your GPU model and GPU driver version? If you roll back to an earlier version, will it work?

**Comment by mcornetto** [comment]
@semjon00 here's the info and clearing the GPU cache did not fix the problem.  I did not install a NVidia driver recently, but it looks like there was some new NVIDIA stuff installed recently (a few NSight apps). 

DirectX version:	12.0 
GPU processor:		NVIDIA GeForce GTX 1660 Ti
Driver version:		560.94
Driver Type:		DCH
Direct3D feature level:	12_1
CUDA Cores:		1536 
Core clock:		1590 MHz 
Memory data rate:	12.00 Gbps
Memory interface:	192-bit 
Memory bandwidth:	288.05 GB/s
Total available graphics memory:	38806 MB
Dedicated video memory:	6144 MB GDDR6
System video memory:	0 MB
Shared system memory:	32662 MB
Video BIOS version:	90.16.4D.00.2A

[Components]

nvui.dll		8.17.15.6094		NVIDIA User Experience Driver Component
nvxdplcy.dll		8.17.15.6094		NVIDIA User Experience Driver Component
nvxdbat.dll		8.17.15.6094		NVIDIA User Experience Driver Component
nvxdapix.dll		8.17.15.6094		NVIDIA User Experience Driver Component
NVCPL.DLL		8.17.15.6094		NVIDIA User Experience Driver Component
nvCplUIR.dll		8.1.940.0		NVIDIA Control Panel
nvCplUI.exe		8.1.940.0		NVIDIA Control Panel
nvWSSR.dll		32.0.15.6094		NVIDIA Workstation Server
nvWSS.dll		32.0.15.6094		NVIDIA Workstation Server
nvViTvSR.dll		32.0.15.6094		NVIDIA Video Server
nvViTvS.dll		32.0.15.6094		NVIDIA Video Server
nvLicensingS.dll		6.14.15.6094		NVIDIA Licensing Server
nvDevToolSR.dll		32.0.15.6094		NVIDIA Licensing Server
nvDevToolS.dll		32.0.15.6094		NVIDIA 3D Settings Server
nvDispSR.dll		32.0.15.6094		NVIDIA Display Server
nvDispS.dll		32.0.15.6094		NVIDIA Display Server
PhysX		09.23.1019		NVIDIA PhysX
NVCUDA64.DLL		32.0.15.6094		NVIDIA CUDA 12.6.65 driver
nvGameSR.dll		32.0.15.6094		NVIDIA 3D Settings Server
nvGameS.dll		32.0.15.6094		NVIDIA 3D Settings Server

**Reply by sonakravtsenko3** [reply]
It is probable that some newly installed stuff is responsible. I hope you would find a solution, please post if you do, it would help others, too.
